src/main/webapp/app/shared/util/firebase/firebase-utils.tsx

```diff
@@ -877,3 +878,22 @@ export function findNestedUuids(obj: any, uuids: string[] = []) {
 
   return uuids;
 }
+
+export function areCancerTypeArraysEqual(a: CancerType[], b: CancerType[]) {
```

Comment:
are these two sorted list?

Reply:
They are not sorted, so if ordering changes, then this tumor name will appear in review mode.